timemachine/ff/__init__.py

```diff
@@ -114,6 +121,7 @@ def from_handlers(
                 lj_handle = handle
             elif (
                 isinstance(handle, nonbonded.AM1CCCHandler)
+                or isinstance(handle, nonbonded.AM1CCCSplitHandler)
                 or isinstance(handle, nonbonded.AM1BCCHandler)
                 or isinstance(handle, nonbonded.SimpleChargeHandler)
```

Comment:
Question 1: Is there a convenient way to allow "split" charges but with the other charge models (AM1BCC, SimpleCharge), or a user-specified function that can assign charges to a ligand?

Question 2: Is there any functionality in common between "splitting" charge parameters (this PR) and "splitting" LJ parameters (not this PR)?

Reply:
1) Updated so all of these charge models have a corresponding intramolecular version. It should be simple enough to add additional handlers if needed.
2) The code should follow the same pattern for LJ.